01.general/15.dns-filtering/docs.en.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "DNS filtering"
+taxonomy:
+    category:
+        - docs
+published: true
+visible: true
+---
+
+* [What is DNS?](#what-is-dns)
+* [How does DNS filtering work?](#dns-filtering)
+    * [DNS servers](#dns-servers)
+    * [Local DNS blocklists](#dns-blocklists)
+* [Comparing DNS filtering to network filtering](#compare)
+
+To better understand DNS filtering, first we should answer the question "What is DNS"?
+
+<a id="what-is-dns"></a>
+### What is DNS?
+
+DNS stands for "Domain name system", and its purpose is to translate websites' names into something browsers can understand, i.e. IP addresses. Thus, each time you go to a website, your browser sends a request to a special server (DNS server). That server looks at the requested domain name and replies with a corresponding IP address. Very schematically it can be represented like this:
```

Comment:
есть ощущение, что 'Very schematically it can be represented like this' похоже на кальку с русского
может, так переписать?
- This might be summed up as follows
- The scheme below shows how DNS works
- Here is a short outline of this